### ISSUE TITLE
Реализовать автоматическое масштабирование фото

### ISSUE THREAD

**Issue by Aleksei-Pankratev-EPAM** [Aleksei-Pankratev-EPAM/CherryPeakBidding#41]
Пользователь может загружать фотографии до 5 МБ размером, что слишком много для использования в качестве миниатюр, да и в качестве полного изображения тоже многовато.

Необходимо реализовать логику масштабирования изображений перед загрузкой в хранилище под 2 размера:
* "полный" размер, для отображения на странице с детализацией лота - ограничения размера обсудить с @Iana-Vdovina-EPAM 
* миниатюра - для отображения в списке лота - оптимальный размер выбрать с @Sergei-Antonov-EPAM 

NB. System.Drawing.Common хотя теперь и кроссплатформенная, на Linux работает с ограничениями. Возможно, имеет смысл сразу выбрать альтернативную библиотеку.

**Comment by Sofia-Vaks-EPAM** [comment]
640*480 - полный размер

**Comment by Sofia-Vaks-EPAM** [comment]
Для корректного использования System.Drawing на Linux необходимо выполнить следующие команды
`sudo apt install libc6-dev 
sudo apt install libgdiplus`

Но рекомендуют использовать библиотеку ImageSharp

**Comment by Pankraty** [comment]
Да, но например в aws lambda это невозможно сделать. System.Drawing обычно рекомендуют только если надо мигрировать существующий код с минимальными переделками в .Net Core. Так что я за ImageSharp

**Comment by Aleksei-Pankratev-EPAM** [comment]
Да, но например в aws lambda это невозможно сделать. System.Drawing обычно рекомендуют только если надо мигрировать существующий код с минимальными переделками в .Net Core. Так что я за ImageSharp

Опыть не из того профиля ответил 😆
